internal/metrics/metrics.go

```diff
@@ -28,6 +28,13 @@ type MetricsService interface {
 	ObserveDBQueryDuration(queryType, table string, duration float64)
 	IncDBQuery(queryType, table string)
 	IncSignatureVerificationExpired(expiredSeconds float64)
+	// Ingestion Phase Metrics
+	ObserveIngestionPhaseDuration(phase string, duration float64)
+	IncIngestionLedgersProcessed(count int)
+	IncIngestionTransactionsProcessed(count int)
+	ObserveIngestionBatchSize(size int)
+	ObserveIngestionParticipantsCount(count int)
+	ObserveIngestionExistingAccountsCount(count int)
```

Comment:
I dont think this existing accounts metric makes sense for ingestion since we already track the accounts count using the `IncActiveAccount` and `DecActiveAccount` functions. We can remove this particular one from ingestion metric